api/v1/azureappconfigurationprovider_types.go

```diff
@@ -53,7 +53,7 @@ type AzureAppConfigurationProviderStatus struct {
 // RefreshStatus defines last refresh time of configmap and secret when dynamic feature is enabled
 type RefreshStatus struct {
 	LastKeyVaultReferenceRefreshTime metav1.Time `json:"lastKeyVaultReferenceRefreshTime,omitempty"`
-	LastSentinelBasedRefreshTime     metav1.Time `json:"lastSentinelBasedRefreshTime,omitempty"`
+	LastKeyValueRefreshTime          metav1.Time `json:"lastKeyValueRefreshTime,omitempty"`
```

Comment:
Please test the upgrade scenario to see whether the field change in CRD would break upgrade scenario, You can try install 1.3.1 first then upgrade to this version to see if it block anything.